reinforcement_learning/agents/common/lazy_tabular_action_value.py

```diff
@@ -101,20 +81,3 @@ def _repr_svg_(self):
 
     def view(self):
         return self._get_graph().view()
-
-if __name__ == '__main__':
-    # SimpleAction-value test
-    av = LazyTabularActionValue()
-
-    s = SimpleState([[-1, -1], [-1, 1]])
-
-    a1 = SimpleAction([0, 0])
-    a2 = SimpleAction([0, 1])
-    a3 = SimpleAction([1, 0])
-
-    av[s, a1] = 6
-    av[s, a2] = 0.8
-    av[s, a3] = -10
-
-    print(av)
-    av.view()
```

Comment:
It's also necessary, because only human-eay can test if graphical appearience is good enaugh...